zips/zip-0317.rst

```diff
@@ -310,26 +324,34 @@ calculated in zatoshis per the following formula:
      zsa\_logical\_actions  &=& logical\_actions \;+ \\
                        & & nTotalOutputsZsaIssuance \\
      zsa\_conventional\_fee &=& marginal\_fee \cdot \mathsf{max}(grace\_actions, zsa\_logical\_actions) \;+ \\
-                       & & issuance\_fee \cdot nIssueActions \;+ \\
-                       & & finalize\_fee \cdot nFinalizeActions
+                       & & issuance\_fee \cdot nCreateActions
    \end{array}
 
 The inputs to this formula are taken from transaction fields defined in the Zcash protocol
-specification [#protocol-txnencoding]_. They are defined in the `Fee calculation`_ section above, with additional definitions in the table below. Note that :math:`nOrchardActions` is redefined and now combines both the Orchard actions for ZEC values as well as ZSA transfer actions for transfer of Assets. 
+specification [#protocol-txnencoding]_ and the global . They are defined in the `Fee calculation`_ section above, with additional definitions in the table below. 
+Note that :math:`nOrchardActions` is redefined and now combines the actions for native ZEC as well as ZSA transfer actions for Custom Assets. 
 
-================================ ====== ========================================================================
+================================ ====== =============================================================================================================
 Input                            Units  Description
-================================ ====== ========================================================================
-:math:`nOrchardActions`          number the number of ZSA transfer actions (including ZEC actions)
+================================ ====== =============================================================================================================
+:math:`nOrchardActions`          number the number of OrchardZSA transfer actions (including ZEC actions)
 :math:`nTotalOutputsZsaIssuance` number the total number of ZSA issuance outputs (added across issuance actions)
-:math:`nIssueActions`            number the number of ZSA issuance actions
-:math:`nFinalizeActions`         number the number of ZSA issuance actions that set the ``finalize`` flag
-================================ ====== ========================================================================
+:math:`nCreateActions`           number the number of ZSA issuance actions that issue a Custom Asset that is not present in the Global Issuance State
```

Comment:
Let's stick to `OrchardZSA` for consistency.

Reply:
Done